code-of-conduct.md

```diff
@@ -5,25 +5,43 @@ and welcoming space for everyone at all times.
 The most important rule is: be excellent to each other.
 
 Sadly, technology communities have a notorious reputation for harassment.
-We've set these ground rules to make it clear that this behaviour is unacceptable
+We've set these ground rules to clarify that this behaviour is unacceptable
 so everyone can feel comfortable being part of Labitat.
 
 Everyone who visits Labitat must abide by this code of conduct,
-both in the physical space and on any online channels.
-We will not tolerate harassment of participants or discriminatory behaviour of
-any form.
+both in the physical space and in any space that utilizes Labitat’s brand,
+e.g., in camps, festivals, etc., and on any online channels.
+
+Members who do not donate the minimum amount for door access
+have access to Labitat when others let them in.
+See [door access](https://labitat.dk/wiki/Door_access) for the minimum donation required.
+Paying members have access to the Labitat door every day via a personal code.
+Sharing your personal code or using someone else's is a violation
+of the code of conduct.
+
+Machines that require training must only be used after an introduction from
+a super user of that machine, even if you have used a similar machine before.
+See the wiki with a [list of super users][who].
```

Comment:
Hafnium notes that this is a nice change, but slightly different from practice so far. We'll need to create posters to make it absolutely clear that you'll still have to talk to a super user even you've used a similar machine before.

Reply:
This was rejected at the 2025-02-04 meeting.